src/app/CommandSender.cpp

```diff
@@ -224,6 +224,9 @@ CHIP_ERROR CommandSender::OnMessageReceived(Messaging::ExchangeContext * apExcha
     {
         err = ProcessInvokeResponse(std::move(aPayload), moreChunkedMessages);
         SuccessOrExit(err);
+#if CONFIG_BUILD_FOR_HOST_UNIT_TEST
```

Comment:
This should not be on "build for host unit test" only.  It could be added at all times. Can store as a uint16_t for RAM usage, but use size_t in API as you have it.